components/FlexibleForms/Banner.tsx

```diff
@@ -0,0 +1,14 @@
+interface Props {
+  title: string;
+  children: React.ReactChild | React.ReactChild[];
+  className?: string;
+}
+
+const Banner = ({ title, children, className }: Props): React.ReactElement => (
+  <section role="alert" className={`lbh-page-announcement ${className}`}>
```

Comment:
Do we use the `classnames` `cx()` function for this type of interpolation?

```ts
cx("lbh-page-announcement", className)
```

Reply:
Because `className` is optional (as per your `Props`) you can end up with `"undefined"` in your class list. Best to avoid that if we can, it can get messy. `classnames` prevents that by automatically filtering them out

Reply:
done